### PR TITLE
Mutt sidebar patch

### PR DESCRIPTION
**Pull request by ephexeve**
Mutt with sidebar patch.

### CODE REVIEW

File: Library/Formula/mutt.rb [hunk 1]
```diff
@@ -72,6 +73,11 @@ class Mutt < Formula
     sha1 "94da52d50508d8951aa78ca4b073023414866be1"
   end if build.with? "confirm-attachment-patch"
 
+  patch do
+    url "http://lunar-linux.org/~tchan/mutt/patch-1.5.23.sidebar.20140412.txt"
+    sha1 "8b86f148d8e1a0b37b0b77268349f4c94e797561"
```
Comment (DomT4) [inline Library/Formula/mutt.rb]: This should be `sha256`.

File: Library/Formula/mutt.rb [hunk 2]
```diff
@@ -72,6 +73,11 @@ class Mutt < Formula
...
+  end if build.with? "sidebar-patch"
```
Comment (DomT4) [inline Library/Formula/mutt.rb]: I know the rest of the formula is styled differently, but we're migrating away from `end if build.with? "sidebar-patch"` and such.

Should be:

```ruby
if build.with? "sidebar-patch"
  patch do
    url "http://lunar-linux.org/~tchan/mutt/patch-1.5.23.sidebar.20140412.txt"
    sha256 "1234"
  end
end
```

Thanks!

Comment (ephexeve) [inline Library/Formula/mutt.rb]: No worries. How about now?

Comment (DomT4) [inline Library/Formula/mutt.rb]: Huge :heart: for changing them all. Thank you!

Comment (ephexeve) [inline Library/Formula/mutt.rb]: Anytime ;-)

**Comment by DomT4** [comment]
There's a little whitespacing on the lines in-between the patches apparently:

```
mutt:
 * 69: Trailing whitespace was found
 * 76: Trailing whitespace was found
 * 83: Trailing whitespace was found
 * 90: Trailing whitespace was found
```

**Comment by ephexeve** [comment]
@DomT4, yup, forgot to do an audit before. Fixed now.

**Comment by DomT4** [comment]
There's a comment at the top of the formula about not adding new patches to this formula, and this patch has been removed before, but there's also a decent chunk of people wandering around online asking how to install the patch on OS X.

The patch here was removed previously because it wasn't getting regular updates on new `mutt` releases and would just break. Has that changed?

CC @Homebrew/owners for Mutt patching opinions.

**Comment by tdsmith** [comment]
I think the text at the head of the formula is clear. If someone wants to set up a patchy-mutt tap, that would be a useful service to the mutt community.

Thanks for the PR; let us know if you have any questions about setting up a tap. In the simplest case, you can put `mutt.rb` in a Github repository named `homebrew-patchy-mutt` or similar (`homebrew-` is important) and then install it with e.g. `brew install ephexeve/patchy-mutt/mutt`.

**Comment by ephexeve** [comment]
@DomT4, well, afaik the sidebar-patch only works with mutt 1.5.23 (which is the version Homebrew is using for sometime now). Archlinux AUR also has [mutt-sidebar](https://aur.archlinux.org/packages/mutt-sidebar/) which is also using 1.5.23

MacPorts has ```mutt-devel``` which ships with mutt-sidebar as a variant.

If that comment is still valid or not, I don't know.

**Pull request closed without merging**